src/vaadin-radio-group.html

```diff
@@ -470,13 +502,43 @@
 
         /** @private */
         _labelChanged(label) {
-          if (label) {
-            this.setAttribute('has-label', '');
+          this._setOrToggleAttribute('has-label', label === 0 || !!label);
+        }
+
+        /** @private */
+        _errorMessageChanged(errorMessage) {
+          this._setOrToggleAttribute('has-error-message', errorMessage === 0 || !!errorMessage);
+        }
+
+        /** @private */
+        _helperTextChanged(helperText) {
+          this._setOrToggleAttribute('has-helper', helperText === 0 || !!helperText);
```

Comment:
Why do we check for 0 here, is it a special value?

Reply:
Done.